Makefile

```diff
@@ -62,5 +62,8 @@ run-e2e-tests: start-e2e-test-env
 	go test -v -count=1 -race ./e2e/...
 	make stop-e2e-test-env
 
+start-local-tools:
+	COMPOSE_BAKE=true docker compose -p treenq -f docker-compose.local.yaml up -d --build
```

Comment:
you can skip build flag I think.
if you wish you can also do `-f docker-compose.yaml -f docker-compose.local.yaml` instead